CONTRIBUTING.md

```diff
@@ -1,51 +1,147 @@
+- `ci`: changes to CI configuration files and scripts (example scopes: travis)
+- `perf`: a code change that improves performance
+- `style`: changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+
+`scope` is optional, and can be anything.
+`description` should be a short description of the change, written in the imperative-mood.
+
+### Example workflow
+
+Follow this process if you'd like your work considered for inclusion in the
+project:
+
+1. [Fork](http://help.github.com/fork-a-repo/) the project, clone your fork,
+   and configure the remotes:
+
+   ```bash
+   # Clone your fork of the repo into the current directory
+   git clone git@github.com:<YOUR_USERNAME>/imgix-python.git
+   # Navigate to the newly cloned directory
+   cd imgix-python
+   # Assign the original repo to a remote called "upstream"
+   git remote add upstream https://github.com/imgix/imgix-python
+   ```
+
+2. If you cloned a while ago, get the latest changes from upstream:
+
+   ```bash
+   git checkout <dev-branch>
+   git pull upstream <dev-branch>
+   ```
+
+3. Create a new topic branch (off the main project development branch) to
+   contain your feature, change, or fix:
+
+   ```bash
+   git checkout -b <topic-branch-name>
+   ```
+
+4. Commit your changes in logical chunks. Use Git's
+   [interactive rebase](https://help.github.com/articles/interactive-rebase)
+   feature to tidy up your commits before making them public.
+
+5. Locally merge (or rebase) the upstream development branch into your topic branch:
+
+   ```bash
+   git pull [--rebase] upstream <dev-branch>
+   ```
+
+6. Push your topic branch up to your fork:
+
+   ```bash
+   git push origin <topic-branch-name>
+   ```
+
+7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
+   with a clear title and description.
+
+**IMPORTANT**: By submitting a patch, you agree to allow the project owner to
+license your work under the same license as that used by the project.
+
+## Test
 
 First, it's recommended you create a virtual environment:
+
 ```
 python -m venv env/imgix
 source env/imgix/bin/activate
 ```
 
 Then install the dev dependencies:
+
 ```
 pip install .[dev]
 ```
 
 To run the tests:
+
 ```
 tox
 ```
+
 This will run all tests and check PEP8 compliance. Our test suite runs
 continuously on [Travis CI](https://travis-ci.org/imgix/imgix-python) with
 every pull request.
 
 To run tests in a specific environment:
+
 ```
 tox -e core  # Your local Python version
 tox -e flake8  # To run the linter
 ```
 
-Publishing to PyPI
-------------------
+## Release
 
-To publish a new version of the package to PyPI, increment the version in [imgix/__init__.py](https://github.com/imgix/imgix-python/blob/main/imgix/__init__.py) run the following:
+To publish a new version of the package to PyPI, increment the version in [imgix/**init**.py](https://github.com/imgix/imgix-python/blob/main/imgix/__init__.py) run the following:
```

Comment:
This might be a typo?
```suggestion
To publish a new version of the package to PyPI, increment the version in [imgix/__init__.py](https://github.com/imgix/imgix-python/blob/main/imgix/__init__.py) run the following:
```

Reply:
Ah 🤦🏼 that's the MD auto formatter running wild. Nice catch.